src/test/java/jenkins/plugins/office365connector/workflow/Office365ConnectorWebhookNotifierTest.java

```diff
@@ -168,22 +167,22 @@ private void setWebhookToJob(List<Webhook> webhooks) {
     }
 
     @Test
-    public void extractWebhooks_OnMissingProperty_ReturnsEmptyWebhooks() {
+    public void extractWebhooks_OnMissingProperty_ReturnsEmptyWebhooks() throws Throwable {
 
         // given
         Job job = mock(Job.class);
         when(job.getProperty(WebhookJobProperty.class)).thenReturn(null);
         Office365ConnectorWebhookNotifier notifier = new Office365ConnectorWebhookNotifier(run, taskListener);
 
         // when
-        List<Webhook> webhooks = Deencapsulation.invoke(notifier, "extractWebhooks", job);
+        List<Webhook> webhooks = MethodReflection.invokeWithCheckedThrows(notifier.getClass(), notifier, "extractWebhooks", new Class[]{Job.class}, job);
 
         // then
         assertThat(webhooks).isEmpty();
     }
 
     @Test
-    public void extractWebhooks_OnMissingWebhooks_ReturnsEmptyWebhooks() {
+    public void extractWebhooks_OnMissingWebhooks_ReturnsEmptyWebhooks() throws Throwable {
```

Comment:
worse than just Exception

Reply:
Yes, but required by the new version of JMockit :man_shrugging: